net/siproxd/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		siproxd
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		Siproxd is a proxy daemon for the SIP protocol
```

Comment:
Maybe shorten this to "Proxy daemon for the SIP protocol"

Reply:
SIP means session initiation protocol :P